demos/text-field.html

```diff
@@ -136,28 +136,28 @@
       </section>
       <section class="example">
         <h2>Password field with validation</h2>
-        <div class="mdc-textfield">
-          <input required pattern=".{8,}" type="password" class="mdc-textfield__input" id="pw"
+        <div class="mdc-text-field">
+          <input required pattern=".{8,}" type="password" class="mdc-text-field__input" id="pw"
                  aria-controls="pw-validation-msg"
                  autocomplete="current-password">
-          <label for="pw" class="mdc-textfield__label">Choose password</label>
-          <div class="mdc-textfield__bottom-line"></div>
+          <label for="pw" class="mdc-text-field__label">Choose password</label>
+          <div class="mdc-text-field__bottom-line"></div>
         </div>
-        <p class="mdc-textfield-helptext mdc-textfield-helptext--persistent mdc-textfield-helptext--validation-msg"
+        <p class="mdc-text-field-helptext mdc-text-field-helptext--persistent mdc-text-field-helptext--validation-msg"
            id="pw-validation-msg">
           Must be at least 8 characters long
         </p>
       </section>
       <section class="example">
-        <h2>Textfield box</h2>
+        <h2>TextField box</h2>
```

Comment:
Should we be referring to this as "Text field" in docs, to match the [guidelines](https://material.io/guidelines/components/text-fields.html)?